sim/visuals/controls/colorGrid.ts

```diff
@@ -33,12 +33,19 @@ namespace pxsim.visuals {
             }
 
             const whiteCircleWrapper = pxsim.svg.child(this.group, "g", { 'id': 'white-cirlce-wrapper' });
-            const circle = pxsim.svg.child(whiteCircleWrapper, "circle", { 'class': 'sim-color-grid-circle sim-color-grid-white', 'cx': 2.2, 'cy': '16', 'r': '2', 'style': `fill: #fff` });
-            this.colorDivs.push(circle);
-            pxsim.svg.child(whiteCircleWrapper, "circle", { 'cx': 2.2, 'cy': '16', 'r': '2', 'style': `fill: none;stroke: #94989b;stroke-width: 0.1px` });
+            const noneCircleWrapper = pxsim.svg.child(this.group, "g", { 'id': 'nothing-cirlce-wrapper' });
+            const whiteCircle = pxsim.svg.child(whiteCircleWrapper, "circle", { 'class': 'sim-color-grid-circle sim-color-grid-white', 'cx': 2.2, 'cy': '16', 'r': '2', 'style': `fill: #fff` });
+            const noneCircle = pxsim.svg.child(noneCircleWrapper, "circle", { 'class': 'sim-color-grid-circle sim-color-grid-none', 'cx': 7.5, 'cy': '16', 'r': '2', 'style': `fill: #fff; fill-opacity: 0%;` });
+            this.colorDivs.push(whiteCircle);
+            this.colorDivs.push(noneCircle);
+            pxsim.svg.child(whiteCircleWrapper, "circle", { 'cx': 2.2, 'cy': '16', 'r': '2', 'style': `fill: none; stroke: #94989b; stroke-width: 0.1px` });
+            pxsim.svg.child(noneCircleWrapper, "circle", { 'cx': 7.5, 'cy': '16', 'r': '2', 'style': `fill: none; stroke: #94989b; stroke-width: 0.1px` });
             pointerEvents.down.forEach(evid => whiteCircleWrapper.addEventListener(evid, ev => {
                 this.setColor(6);
             }));
+            pointerEvents.down.forEach(evid => noneCircleWrapper.addEventListener(evid, ev => {
+                this.setColor(0);
```

Comment:
To confirm, this is `this.setColor(0)` because it corresponds to `this.setColor(ColorGridControl.colorValue[7])`?

Reply:
Um, color 7 is brown. 0 is no color, 1 is black, 6 is white. So according to the Lego table. But, I noticed that in the code brown is called gray for some reason. Just noticed.

Reply:
Right, I meant that the it matched to the value in here https://github.com/microsoft/pxt-ev3/pull/1018/files#diff-e2e36c2a459faefa8f551f5d676fbfca2d877fe0bae486913114c0540a68c4c8R8-R9 matching the value at the index where "none" is stored. Seems good, not sure why brown was called gray either though~